test/selenium/conf/selenium-test.conf

```diff
@@ -7,10 +7,12 @@ identity {
   test.users.secret = ${?IDENTITY_TEST_USERS_SECRET}
 }
 support.url = ${?SUPPORT_URL}
+contribution.url = ${?CONTRIBUTION_URL}
 web.driver.remote.url = ${?WEBDRIVER_REMOTE_URL}
 
 // Uses details from support-frontend.private.conf when running locally
 paypal.sandbox {
+  url = ${?PAYPAL_SANDBOX_URL}
```

Comment:
The PayPal sandbox url is not private, and it's the same regardless of whether we are running these tests against local builds or PROD builds. Consequently I don't think we need to use an environment variable here (just putting the url in this conf file should work when running locally and for Travis builds).